Trap Quest.inform/Inform/Extensions/Every Turn/Periodic Effects.i7x

```diff
@@ -207,6 +207,54 @@ To compute stat healing:
 		decrease temp_int_dam by 1;
 		say "[bold type]You feel as though your mind is slightly clearer![roman type]";
 
+A time based rule (this is the audible jiggling rule):
+	if a random number between 1 and 100 < 50 + unlucky:
+		if debugmode > 0, say "AUDIBLE CHECK. ";
+		let audibly jiggling clothing be nothing;
+		repeat with C running through worn cursed blandness clothing:
+			if C is overdress or C is bikini top or C is bra:
+				If C is chestless or fully exposing, decide on 0;
+				If the largeness of breasts <= 2,decide on 0;
+				if C is overdress and support of (C - a bra) is >= 2, decide on 0;
```

Comment:
Is this `decide on 0;` something you've learned to do elsewhere to break from a block early or something? I can imagine it might work, I've just never tried it before.

Reply:
I saw it used in some of the other files here. I thought it acted similarly to a type of break statement that I know from my rudimentary knowledge of Java.

Reply:
I use it when the block is a function that returns a number, i.e. starts with `To decide which number is`.

I don't think it's particularly necessary to do this here especially because it might have weird unexpected functionality. There are lots of other ways to use if statements and logic and indentation to make sure that only the desired items reach the next bit of code.